typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -590,26 +592,25 @@ async function getNativeTokenWarpInfo(
 }
 
 async function getCollateralTokenPrice(
-  tokenCoinGeckoId: string | undefined,
+  tokenCoingeckoId: string | undefined,
   tokenPriceGetter: CoinGeckoTokenPriceGetter,
 ): Promise<number | undefined> {
-  if (!tokenCoinGeckoId) return undefined;
-  const prices = await tokenPriceGetter.getTokenPriceByIds([tokenCoinGeckoId]);
-  logger.debug(`${tokenCoinGeckoId} prices ${prices}`);
+  if (!tokenCoingeckoId) return undefined;
+  const prices = await tokenPriceGetter.getTokenPriceByIds([tokenCoingeckoId]);
   if (!prices) return undefined;
   return prices[0];
 }
 
 async function getCollateralTokenValue(
-  tokenCoinGeckoId: string | undefined,
+  tokenCoingeckoId: string | undefined,
   balanceFloat: number,
   tokenPriceGetter: CoinGeckoTokenPriceGetter,
 ): Promise<number | undefined> {
-  logger.debug('tokenCoinGeckoId', tokenCoinGeckoId);
   const price = await getCollateralTokenPrice(
-    tokenCoinGeckoId,
+    tokenCoingeckoId,
     tokenPriceGetter,
   );
+  logger.debug(`${tokenCoingeckoId} token price ${price ? price : undefined}`);
```

Comment:
nit: could you just use `price` directly instead of `price ? price : undefined`?